apps/feedahelp/package.json

```diff
@@ -40,7 +40,7 @@
     "react-hook-form": "^7.45.3",
     "react-hot-toast": "^2.4.1",
     "recoil": "^0.7.7",
-    "styled-components": "^6.0.7",
+    "styled-components": "^5.2.0",
```

Comment:
you downgraded this, but why?

Reply:
cz next-sanity package showed a warning to downgrade that

Reply:
ok
lets see